samples/docker/README.md

```diff
@@ -4,6 +4,48 @@
 
 The following instructions detail how to build and run the FHIR Server in Docker on Linux.
 
+## Use CI image
+
```

Comment:
Do we plan to provide "Deploy to Azure" button using the containers?

Reply:
We already have one here: https://github.com/microsoft/fhir-server/blob/master/docs/DefaultDeployment.md#deploying-the-fhir-server-template
The next story (#72981) will update the deployment script to use the container.